api/v1/loadtest_types.go

```diff
@@ -206,6 +206,12 @@ type LoadTestSpec struct {
 	// Scenarios provides a list of configurations for testing.
 	// +optional
 	Scenarios []Scenario `json:"scenarios,omitempty"`
+
+	// Timeout provides the longest running time allowed for a LoadTest.
+	TimeoutSeconds int32 `json:"timeoutSeconds"`
+
+	//TTL provide the longest time a LoadTest could live on the luster.
```

Comment:
I think there may be typos here, but otherwise good. 

```suggestion
	// TTL provide the longest time a LoadTest could live on the cluster.
```